### PR TITLE
Add weekly build

### PR DESCRIPTION
**Pull request by sue445**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ on:
       - opened
       - synchronize
       - reopened
+  schedule:
+    - cron: "0 0 * * 5" # JST 9:00 (Sat)
```
Comment (sue445) [inline .github/workflows/test.yml:13]: **[Consultation]** This setting is sloppy :tired_face:
Do you have any good idea?

Comment (unasuke) [reply .github/workflows/test.yml:13]: What is this "sloppy"? It looks like no problem... Could you explain how is this "sloppy"? 👀

Comment (sue445) [reply .github/workflows/test.yml:13]: I want to run a scheduled build once a week.
But I can't think of a good schedule. (e.g. week, time)

Comment (unasuke) [reply .github/workflows/test.yml:13]: Hmm... ☺️

**Review by unasuke — APPROVED** [review]
🙆

**Pull request merged**